docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
I think that may be from a "best practice" perspective. It's generally discouraged to tag your local images with the "username/name:tag" format, because doing so would overwrite your local images when you `docker pull` that same image from the repository. Omitting `username/` for your local images makes it easier to distinguish your locally built images from the ones you pulled.

(Just my thoughts why it was written that way)